src/collections/_documentation/platforms/javascript/angular.md

```diff
@@ -47,6 +105,48 @@ export class SentryErrorHandler implements ErrorHandler {
 export class AppModule {}
 ```
 
+When using your own `ErrorHandler`, make sure that whenever you use `HttpInterceptor` alongside it,
+it doesn't modify the error captured originally.
+The same goes for writing your own API Services with built-in `http` methods.
+
+For example, the service below make it impossible for the SDK to extract correct value.
+The reason being that it's not passed down the error pipeline in it's entirety, but returning `error` property of the original instead.
```

Comment:
```suggestion
For example, the service below makes it impossible for the SDK to extract the correct data, because the full, original error `e` is not propagated.
```